### ISSUE TITLE
Tables suddenly stopped displaying properly

### ISSUE THREAD

**Issue by LixyHeart** [tgrosinger/advanced-tables-obsidian#262]
Once I create a table under text it doesn't format properly. This has changed all tables from yesterday leaving them unreadable now.

1. Go to a New note
2. Type text then pressed enter
3. Shift + \ to get | whilst typing table contents using tab and enter like usual
4. Click off of table 
5. See error

I expected for it to work as normal formatting the table like the above example in the screenshot. No clue why its stopped working so suddenly.

![Screenshot (15)](https://github.com/tgrosinger/advanced-tables-obsidian/assets/134787374/807a11fe-961b-4234-9600-46932e1fc6de)

![Screenshot (16)](https://github.com/tgrosinger/advanced-tables-obsidian/assets/134787374/18adc54d-0c45-4b22-a874-91874e3ebbe1)

 - Windows 10
 - Obsidian Version: v1.3.4
 - Advanced Tables Version: 0.18.1

This hasn't been an issue until yesterday but I've been getting the obsidian update tab before that so i don't think the update did it. Using obsidian for a dnd campaign so this is a lot of work destroyed by this bug. Apologies if I'm a little lost since this is my first GitHub post thank you for reading.

**Comment by khorton** [comment]
Is there perhaps a blank line between the text "Working table" and the table, and no blank line between "Bug" and the broken table?

Apparently, Markdown tables require a blank line immediately before the table.  Obsidian won't preview the table correctly unless there is a blank line immediately above the table.

**Comment by LixyHeart** [comment]
> Is there perhaps a blank line between the text "Working table" and the table, and no blank line between "Bug" and the broken table?
> 
> Apparently, Markdown tables require a blank line immediately before the table. Obsidian won't preview the table correctly unless there is a blank line immediately above the table.

yes im pretty sure this is the case but my confusion comes from this exact requirement being non existent two days ago so im unsure why its suddenly doing it now. I can work round it and provide it the blank line it just looks little worse :<

**Comment by How2PlayGithub** [comment]
Hey, I'm also having the same problem. My old table was displaying the table fine until a few days ago. You just need to retype them

**Comment by alan-daniel** [comment]
yeah, I'm also a bit confused as to why this changed. Glad I stumbled onto this, as I was more than a little frustrated that every single one of my tables broke.

Surely there's a way to revert to the behavior from before right now, right? I'd rather not have to add blank lines everywhere, as vertical space is at a premium and I spent a lot of time and energy getting my notes/etc. formatted the way I have.

**Comment by tgrosinger** [comment]
I understand the frustration, but unfortunately this problem is caused by a change in Obsidian and this plugin can't fix it.

Please reach out to the Obsidian developers here:

<https://forum.obsidian.md/t/tables-dont-render-in-live-preview-unless-theres-a-blank-line-above/60239>

**Comment by eggtoastie** [comment]
I'm also on obsidian v1.3.4 and Advanced Tables v0.18.1 (Win11) and am having the same issue. It definitely started right after I updated Obsidian. 

Thank you for pointing out the new need for a blank like above the table. That's an easy work around.

**Comment by LixyHeart** [comment]
> I understand the frustration, but unfortunately this problem is caused by a change in Obsidian and this plugin can't fix it.
> 
> Please reach out to the Obsidian developers here:
> 
> https://forum.obsidian.md/t/tables-dont-render-in-live-preview-unless-theres-a-blank-line-above/60239

Ah this makes me really sad but at least for me it doesnt completely ruin my notes, thank you for the link and awesome plugin!
